docs/src/tutorial.md

```diff
@@ -55,16 +51,18 @@ AgentBasedModel
 
 ## 4. Evolving the model
 
-Any ABM model should have at least one and at most two step functions.
+Any ABM model should have at least one and at most two stepping functions.
```

Comment:
This is really a minor gripe, but "ABM model" is a bit repetitive, is it not? It's like saying OSM map.